examples/eight_schools/main_mcmc.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+import sys
+
+import numpy as np
+import pandas as pd
+import torch
+
+import data
+import pyro
+import pyro.distributions as dist
+import pyro.poutine as poutine
+from pyro.infer import EmpiricalMarginal
+from pyro.infer.mcmc import MCMC, NUTS
+
+
+def model(sigma):
+    eta_dist = dist.Normal(torch.zeros(data.J), torch.ones(data.J))
+    mu_dist = dist.Normal(loc=torch.zeros(1), scale=10 * torch.ones(1))
+    tau_dist = dist.HalfCauchy(loc=torch.zeros(1), scale=25 * torch.ones(1))
+
+    eta = pyro.sample('eta', eta_dist)
+    mu = pyro.sample('mu', mu_dist)
+    tau = pyro.sample('tau', tau_dist)
+
+    theta = mu + tau * eta
+
+    return pyro.sample("obs", dist.Normal(theta, sigma))
+
+
+def conditioned_model(model, sigma, y):
+    return poutine.condition(model, data={"obs": y})(sigma)
+
+
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
```

Comment:
You could just modify the format to `%(message)s` and use this instead of the `print` function. You shouldn't have to force the stream to `sys.stdout` which should be the case by default.